openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -517,8 +517,10 @@ def get_estimate(self):
 
             dGs.append(dG)
             #weights.append(weight)
+            
+        avg_val = np.average([i.value_in_unit(dGs[0].unit) for i in dGs])
```

Comment:
Calling `np.average` on the list of quantities yields a `ValueError: setting an array element with a sequence` (at least with numpy 1.23.4). Doing this is a bit convoluted but avoids the issue.